test/active_record_test.rb

```diff
@@ -94,6 +94,12 @@ def teardown
     assert_equal expected_hash, I18n.t('.')
   end
 
+  test "accessing keys with a leading period" do
+    expected_hash = { :bar => 'bar', :baz => 'baz' }
+    assert_equal expected_hash, I18n.t('foo')
+    assert_equal expected_hash, I18n.t('.foo')
```

Comment:
What about `I18n.t('foo.')`? I think it should also return the same

Reply:
Good catch, I have added support for that.